### DISCUSSION TITLE
how to get bounding box by document

### DISCUSSION THREAD

**Discussion by qiangwai** [donmccurdy/glTF-Transform#529] [Q&A] [answered]
I want to use this SDK to compute gltf bounding-box in node environment，like threejs :
`var bbox = new THREE.Box3().setFromObject(obj);`
I want get this box ,thank you

**Comment by donmccurdy** [comment, marked as answer]
The [`bounds(object)`](https://gltf-transform.donmccurdy.com/functions.html#bounds) function can be used to compute the bounding box (AABB) of a Node or Scene:

```js
import { bounds } from '@gltf-transform/functions';

const root = document.getRoot();
const scene = root.getDefaultScene() || root.listScenes()[0];

const bbox = bounds(scene);
console.log(bbox); // → {min: [-1, -1, -1], max: [1, 1, 1]}
```
